### ISSUE TITLE
Please publish an artifact to Maven Central

### ISSUE THREAD

**Issue by rgoldberg** [zyxist/chainsaw#39]
Please publish a Chainsaw artifact to Maven Central.

I want to use a custom plugin to apply Chainsaw (along with other standard plugins & configs) to all my Java builds.

For that, I think I need a dependency in my `dependencies` block.

If you know some other way to make `com.zyxist.chainsaw.ChainsawPlugin` available to my source, please let me know.

**Comment by mdavis95** [comment]
You might try using jitpack.io:

https://jitpack.io/#zyxist/chainsaw/chainsaw-0.3.1

```gradle
repositories {
    maven { url 'https://jitpack.io' }
}

dependencies {
    compile 'com.github.zyxist.chainsaw:chainsaw:chainsaw-0.3.1'
}
```

**Comment by rgoldberg** [comment]
@mdavis95 Thanks for the suggestion.

I've actually stopped using Chainsaw, and made [my own fork](https://github.com/rgoldberg/gradle-java-modules).

Look at [issues in the original repo](https://github.com/gradle/gradle-java-modules/issues) to see things I've fixed.

All of my changes are in branches, but I've postponed creating most intermediate branches because they get in the way of rebasing.

To see the current state of the code, either use the branch `renamed-plugin` if you want to use a different plugin name, or use the commit immediately before the tip of that branch if you want to use the original plugin name.

I haven't published the current version of it to the Gradle Plugin Portal, because I keep updating it, but I can publish it if will help you; I'm trying to get the Gradle team to incorporate my improvements into their repo, to obviate the need for a separate plugin name…
